pkg/disk/controllerserver.go

```diff
@@ -81,10 +85,38 @@ var veasp = struct {
 
 var delVolumeSnap sync.Map
 
+func newSnapshotStatusWaiter() waitstatus.StatusWaiter[ecs.Snapshot] {
+	client := desc.Snapshots{
+		Client: GlobalConfigVar.EcsClient,
+	}
+	if GlobalConfigVar.DiskMultiTenantEnable {
+		waiter := waitstatus.NewSimple(client, clock.RealClock{})
+		waiter.ClientFactory = func(ctx context.Context) (desc.Client[ecs.Snapshot], error) {
+			tenantUserUID := GetTenantUserUID(ctx)
```

Comment:
This block is no longer needed since whole multi-tenant feature has been removed

Reply:
year, will rebase

Reply:
done